test-requirements.in

```diff
@@ -14,3 +14,13 @@ flake8
 
 # https://github.com/python-trio/trio/pull/654#issuecomment-420518745
 typed_ast;python_version<"3.7" and implementation_name=="cpython"
+
+# Trio's own dependencies
+cffi; os_name == "nt"
```

Comment:
I don't understand why pip-compile adds `contextvars==2.3 ; python_version < "3.7"` correctly to test-requirements.txt, but not this one.